tests/e2e/util.go

```diff
@@ -1815,10 +1815,20 @@ func createResourceQuota(client clientset.Interface, namespace string, size stri
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 	waitTime := 15
+	var executeCreateResourceQuota bool
+	executeCreateResourceQuota = true
 	storagePolicyNameForSharedDatastores := GetAndExpectStringEnvVar(envStoragePolicyNameForSharedDatastores)
 
-	_, err := client.CoreV1().ResourceQuotas(namespace).Get(ctx, namespace+"-storagequota", metav1.GetOptions{})
-	if err != nil || !(scName == storagePolicyNameForSharedDatastores) {
+	if supervisorCluster {
+		_, err := client.CoreV1().ResourceQuotas(namespace).Get(ctx, namespace+"-storagequota", metav1.GetOptions{})
+		if err != nil || !(scName == storagePolicyNameForSharedDatastores) {
+			executeCreateResourceQuota = true
```

Comment:
shouldn't we also check if we have enough quota for the tests here

Reply:
The storage quota of storagePolicyNameForSharedDatastores that is shared-ds-policy , is set to max 50GB in pre-setup. That is not changed in any testcase. Since every tests creates and deletes pvc/pod's quota remains same. Not required check the enough quota available or not. 

Where as there is another storage class that is set to STORAGE_POLICY_FOR_SHARED_DATASTORES_2 , this is used in case of create delete resource quota test cases, for this storage class , it sets executeCreateResourceQuota to true and sets to any quota limit that is passed in the testcase.